app/assets/main/react_components/User/CarouselHeader.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function CarouselHeader() {
```

Comment:
Kanske uppdatera för att hålla konsekvent för samtliga?

```suggestion
const CarouselHeader = () => {
```